README.md

```diff
@@ -18,7 +18,6 @@ Some of my hobbies/interests:
 * Running
    * Heading along the lakeshore in the mornings, give me a shout if you also like running! 
```

Comment:
How safe is it to run by the lakeshore in the mornings? I tried running a few times but it is so hard to be disciplined without a buddy!

Reply:
I usually go around 6a or 7a and it's quite safe, there are plenty of other runners at that hour! I highly recommend it as the view of the sunrise hitting the downtown skyline is quite beautiful. We can form a running group once we're all on campus!